doc/docker.md

```diff
@@ -1,111 +1,163 @@
 
-2. To see which containers are running (and with the optional flag to see stopped containers also):
+Take a look at the Makefile to see all the steps in this make target. You can run these steps individually using the additional make targets provided. It may also be useful to check out what the `make` targets from the Makefile are doing.
 
-        docker ps [-a]
 
- **Note:** docker commands and make targets that launch or stop running the containers may return immediately, but take several seconds to take effect. `docker ps` is super useful for this!
+ **Note:**  `docker stack` commands and make targets that launch or stop running the stack containers may return immediately, but take several seconds to take effect. `docker ps` is super useful for this!
+
+2. Individual targets in the Makefile exist for building each docker image. If making local changes to a single component, you can propagate them to the docker stack by first rebuilding the image with:
 
-3. Individual targets in the Makefile exist for building each docker image. If making local changes to a single component, you can propagate them to the docker stack by first rebuilding the image with:
 
         make <image_name>
 
 
 And then restarting the service to run with the new image:
 
+
         docker service update --force $(docker stack services ls -f name=CONTAINER_NAME)
 
 
-4. While the docker stack is running, it will restart any stopped containers automatically using the most recently built image. To stop and restart the entire stack, use the commands
+3. While the docker stack is running, it will restart any stopped containers automatically using the most recently built image. To stop and restart the entire stack, use the commands
 
         make dev_stop
-        make dev
+        make dev_deploy
 
 Note: the `dev_deploy` target rebuilds every image in the stack which may be time-consuming and not necessary if only updating a single image/service. To start a new docker stack without rebuilding the images, use:
 
         make dev_stop
         make dev_start
 
-5. To stop a single *service* in the running stack, run:
+4. To stop a single *service* in the running stack, run:
 
         docker service rm recordexpungpdx_*
 
-replacing * with the service name e.g.  webserver or expungeservice. This stops the corresponding docker container and also prevents the docker orchestrator from spinning up a replacement container. This is useful if you want to, for example, run the node server for frontend development, since the node daemon automatically and rapidly propagates source code changes.
+replacing * with the service name e.g.  webserver or expungeservice. This stops the corresponding docker container and also prevents the docker orchestrator from spinning up a replacement container.
 
-6. To explore the database:
+5. To explore the database:
```

Comment:
I'd suggest changing this to `To start a PostgreSQL interactive terminal`

And you may want to mention that `\q` exits